http_server/src/main.rs

```diff
@@ -160,16 +169,56 @@ async fn generate_sxg_response(
     Ok(sxg.map(Body::from))
 }
 
-async fn serve_preset_content(url: &str) -> Option<PresetContent> {
+/// Persistent storage mechanism for OCSP responses & ACME certs. Takes a path
+/// to a directory where it will create files for them.
+pub struct FileStorage(PathBuf);
+
+#[async_trait]
+impl Storage for FileStorage {
```

Comment:
I would suggest using an external library (such as [sled](https://crates.io/crates/sled)) in the future, which handles file system edge cases for us.

Reply:
Okay, I'll add a TODO to the next PR (so I don't have to request re-approval here).

I chose something simple and lightweight because our KV store only has two entries, but yeah it's possible I'm missing other file system edge cases.